core_lib/helpers/shell_utils.py

```diff
@@ -0,0 +1,78 @@
+from core_lib.helpers.validation import is_int
+
+
+def check_default(value, default_value):
+    if not value and default_value is not None:
+        return default_value
+    else:
+        return value
+
+
+def input_yes_no(title, default_value: bool) -> bool:
+    converted_default_value = 'yes' if default_value else 'no'
+    user_input = input(f'{title} (yes/no) [default: {converted_default_value}]: ')

```

Comment:
this should be also in the while. it's duplicate code 
(don't call your variable `ask` this is an example)
```Python
ask = True 
while ask: 
    user_input = input(...
    ask = ..... (True if ask again False if we have the value)
```